go.mod

```diff
@@ -358,10 +358,4 @@ require (
 	inet.af/peercred v0.0.0-20210906144145-0893ea02156a // indirect
 )
 
-require (
-	github.com/bep/godartsass/v2 v2.0.0 // indirect
-	github.com/hashicorp/go-plugin v1.4.4 // indirect
-	github.com/hashicorp/terraform-registry-address v0.0.0-20220623143253-7d51757b572c // indirect
-	github.com/hashicorp/terraform-svchost v0.0.0-20200729002733-f050f53b9734 // indirect
-	github.com/oklog/run v1.0.0 // indirect
```

Comment:
🤔 Is this related to your change?

Reply:
I'm concerned too, but if you copy `go.mod` and `go.sum` from main here, and then run `go mod tidy`, this is the result.

Reply:
Perhaps we should add `tidy` as a lint/gen check?

Reply:
Sounds about right, but rather in a separate PR.